gui/gui

```diff
@@ -210,20 +210,22 @@ class NumixFoldersGUI(Gtk.Application):
         self.colourentries = []
         self.colourlabels = []
         for i in range(number_colours):
+            hboxcolour.append(Gtk.HBox(spacing=12))
             self.colourbuttons.append(Gtk.ColorButton())
             self.colourbuttons[i].set_color(Gdk.color_parse(self.colour[i]))
-            self.colourlabels.append(Gtk.Label())
             self.colourentries.append(Gtk.Entry())
             self.colourentries[i].set_text(self.colour[i])
             self.colourentries[i].set_max_length(7)
-            self.colourlabels[i].set_text(colour_labels[i])
-            self.colourentries[i].set_max_width_chars(7)
             self.colourentries[i].set_width_chars(7)
+            if Gtk.get_major_version() >= 3 and Gtk.get_minor_version() >= 12:
+                self.colourentries[i].set_max_width_chars(7)
+                self.colourlabels.append(Gtk.Label())
+                self.colourlabels[i].set_text(colour_labels[i])
```

Comment:
Aren't line 222 and 223 something that is necessary for both cases? (not only if Gtk >=3.12)